tsl/src/compression/compress_utils.c

```diff
@@ -12,6 +12,7 @@
 #include <catalog/dependency.h>
 #include <commands/tablecmds.h>
 #include <commands/trigger.h>
+#include <commands/event_trigger.h>
```

Comment:
Does this do anything here?

Reply:
Nope, left-over from a change I reverted. Will remove.